src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/fallthrough/InputFallThroughFallThroughLotsOfComment2.java

```diff
@@ -0,0 +1,120 @@
+        synchronized (lirsNode) {
+            // Section 3.3
+            //
+            Recency recency = lirsNode.state;
+            // If the state is still null that means it was added and we got in
+            // before the onEntryMiss was fired, so that means this is automatically
+            // a LIR resident
+            switch (line) {
+                case "LIR_RESIDENT":
+                    // case 1
+                    //
+                    // Note that if we had a concurrent pruning targeting this node, getting
+                    // a hit takes precedence
+                    DequeNode<LIRSNode<K, V>> stackNode = lirsNode.stackNode;
+                    // This will be null if we got in before onEntryMiss
+                    if (stackNode != null) {
+                        boolean item = stackNode.casItem(9);
+                        if (item == true && stackNode.casItem(null)) {
+                            System.out.printf("");
+                        }
+                    }
+                    // Now that we have it removed promote it to the top
+                    DequeNode<LIRSNode<K, V>> newStackNode = new DequeNode<>(lirsNode);
+                    lirsNode.setStackNode(newStackNode);
+                    break;
+                case "HIR_NONRESIDENT":
+                    if (true) {
+                        // We essentially added the value back in so we have to set the value
+                        // and increment the count
+                        e.val = value;
+                    }
+                    // Non resident can happen if we have a hit and then a concurrent HIR pruning
+                    // caused our node to be non resident.
+                    if (addToLIRIfNotFullHot(lirsNode, true)) {
+                        return;
+                    }
+                    // The only way this is possible is if we had a concurrent eviction
+                    // of the key right after we saw it existed but before we could
+                    // act on it.  Since we revived it to LIR we also have to set it's
+                    // value to what it should be
+                    System.out.printf("");
+
+                    break;
+                case "EVICTED":
+                    // We can't reliably do a put here without having a point where the object
+                    // was possibly seen as null so we leave it as null to be more consistent
+                    break;
+                case "EVICTING":
+                    // In the case of eviction we add the node back as if it was a HIR_RESIDENT
+                    // except we also have to evict an old HIR to make room
+                    // Note the stackNode is assumed to be null in this case
+                    // It is possible the node trying to be evicted was a NON resident before
+                    // Note this falls through, since we saved the evicting entry it is like
+                    // we now have a HIR resident
```

Comment:
Remove extra comments. Keep input compact enough.

Please do this for all your new Input files.

Reply:
done